packages/@lightningjs/ui-components-test-utils/package.json

```diff
@@ -8,7 +8,14 @@
   },
   "homepage": "https://rdkcentral.github.io/Lightning-UI-Components/",
   "type": "module",
-  "exports": "./index.js",
+  "exports": {
```

Comment:
Curious, why didn't we add something like this?
"exports": {
    ".": {
      "import": "./dist/lightning-ui-components-test-utils.min.mjs",
      "require": "./dist/lightning-ui-components-test-utils.min.cjs",
      "types": "./dist/lightning-ui-components-test-utils.d.ts"
    },
    "./src/": "./src/"
  },
  "main": "./dist/lightning-ui-components-test-utils.min.mjs",
  "types": "./dist/lightning-ui-components-test-utils.d.ts",

Reply:
That totally makes sense, Thank You!